docs/getting-started/install-feast/ibm-cloud-iks-with-helm.md

```diff
@@ -0,0 +1,151 @@
+
+### Overview
+
+This guide installs Feast on an existing IBM Kubernetes cluster, and ensures the following services are running:
+
+* Feast Core
+* Feast Online Serving
+* Postgres
+* Redis
+* Feast Jupyter \(Optional\)
+* Prometheus \(Optional\)
+
+### 1. Requirements
+
+1. [IBM Cloud Kubernetes Service](https://www.ibm.com/cloud/kubernetes-service)
+2. Install [Kubectl](https://cloud.ibm.com/docs/containers?topic=containers-cs_cli_install#kubectl) that matches the major.minor versions of your IKS.
+3. Install [Helm 3](https://helm.sh/)
+
+### 2. IBM Cloud Block Storage Setup
+
+By default, IBM Cloud Kubernetes cluster uses [IBM Cloud File Storage](https://www.ibm.com/cloud/file-storage) based on NFS as the default storage class. By default, non-root users do not have write permission on the volume mount path for NFS-backed storage. Some common container images, such as Redis, Postgres, and Kafka specify a non-root user to access the mount path in the images. When containers are deployed using these images, the containers fail to start due to insufficient permissions of the non-root user creating folders on the mount path.
+
+[IBM Cloud Block Storage](https://www.ibm.com/cloud/block-storage) allows for the creation of raw storage volumes and provides faster performance without the permission restriction of NFS-backed storage 
+
+Therefore, we strongly recommend to set up [IBM Cloud Block Storage](https://cloud.ibm.com/docs/containers?topic=containers-block_storage#install_block) as the default storage class so that you can
```

Comment:
```suggestion
Therefore, we to deploy Feast we need to set up [IBM Cloud Block Storage](https://cloud.ibm.com/docs/containers?topic=containers-block_storage#install_block) as the default storage class so that you can
```